streams/src/test/java/org/apache/kafka/streams/kstream/internals/WindowedStreamPartitionerTest.java

```diff
@@ -138,10 +137,42 @@ public void testWindowedDeserializerNoArgConstructors() {
         props.remove("value.deserializer.inner.class");
         WindowedDeserializer<?> windowedDeserializer1 = new WindowedDeserializer<>();
         windowedDeserializer1.configure(props, false);
-        Deserializer<?> inner1 = windowedDeserializer1.innerDeserializer();
+        final Deserializer<?> inner1 = windowedDeserializer1.innerDeserializer();
         assertNotNull("Inner deserializer should be not null", inner1);
         assertTrue("Inner deserializer type should be ByteArrayDeserializer", inner1 instanceof ByteArrayDeserializer);
         windowedDeserializer.close();
         windowedDeserializer1.close();
     }
+    
+    @Test
+    public void testWindowSerializeExpectedWindowSize() {
+        final long randomLong = 5000000;
+        final Map<String, String> props = new HashMap<>();
+        final WindowedDeserializer<StringSerializer> windowedDeserializer = new WindowedDeserializer<>(randomLong);
+        props.put("key.deserializer.inner.class", "org.apache.kafka.common.serialization.StringDeserializer");
+        props.put("deserializer.inner.class", "org.apache.kafka.common.serialization.StringDeserializer");
```

Comment:
Isn't this property ignored if you set `key.deserializer.inner.class` above already? Guess this line can be removed?